includes/server/class-llms-rest-access-plans-controller.php

```diff
@@ -629,7 +629,7 @@ protected function prepare_object_for_response( $access_plan, $request ) {
 		$context = $request->get_param( 'context' );
 
 		// Price.
-		$data['price'] = $access_plan->is_free() ? 0 : $access_plan->get_price( 'price', array(), 'float' );
+		$data['price'] = $access_plan->is_free() ? 0 : $access_plan->get_price( 'price', array(), 'number' );
```

Comment:
The method doesn't have a `number` option. Using this defaults to no formatting which is perhaps what you're desiring but I'm not sure why

Reply:
No that's just a bad search and replace :(

Reply:
Ah, glad I noticed it in that case!

Reply:
fixed in https://github.com/gocodebox/lifterlms-rest/pull/215/commits/ed4f6b0d5e0b1375052633000564fa20112d7cdd